unime/src/routes/(app)/me/search/+page.svelte

```diff
@@ -21,7 +21,14 @@
     <Search
       on:value={(e) => {
         searchTerm = e.detail;
-        dispatch({ type: '[User Data] Query', target: 'Credentials', search_term: e.detail });
+        dispatch({ 
+          type: '[User Data] Query',
+          payload: {
+            target: 'Credentials',
+            search_term: e.detail,
+            sort_reverse: false,
```

Comment:
Can `sort_reverse` have a default of `false`, so it doesn't need to be explicitly passed?